packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Interaction Code grant type
+meta:
+  - name: description
+    content: An overview of the Interaction Code grant type for Okta Identity Engine.
+---
+
+# Interaction Code grant type
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+## Overview
+
+To enable a more customized user authentication experience, Okta has introduced an addition to the [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid) standard called the Interaction Code grant type. This grant type allows apps to manage user interaction with Okta without a browser. This is useful when the client has a particular way that it wants to interact with the user and doesn’t need to share an authenticated session with other applications.
```

Comment:
"manage user interaction with Okta without a browser" and future text, e.g. "The Interaction Code flow consists of a series of interactions between the user and an Okta org, facilitated by the client" and "The client obtains these remediation steps from the Identity Engine component of the Okta org and prompts the user for the required data to continue the flow." seem to contradict each other.

By "client", we mean "client-side application", don't we? How does this not require a browser? If you want to split hairs you can say "native apps that use Okta don't require a browser", but I think the question still stands. This could use some clarification.